packages/store/addon/-private/system/store/record-data-store-wrapper.ts

```diff
@@ -19,12 +20,10 @@ const MISSING_ID_ARG_ERROR_MESSAGE = `Either an id or a clientId is required as
 
 export default class RecordDataStoreWrapper implements IRecordDataStoreWrapper {
   [BRAND_SYMBOL]: 'RecordDataStoreWrapper';
-  _store: Store;
   _willUpdateManyArrays: boolean;
   private _pendingManyArrayUpdates: StringOrNullOrUndefined[];
 
-  constructor(store: Store) {
-    this._store = store;
+  constructor(public _store: CoreStore) {
```

Comment:
where are we still accessing this outside of the class?

Reply:
https://github.com/emberjs/data/blob/master/packages/store/addon/-private/system/relationships/state/create.ts#L36